Js/index.js

```diff
@@ -4,24 +4,172 @@ let btnCancelMobileMenu = document.querySelector("#btn-close-nav");
 const PortfolioClicked = document.querySelector(".link-portfolio");
 const AboutClicked = document.querySelector(".link-about");
 const ContactClicked = document.querySelector(".link-contact");
+const allBtnPopups = document.querySelectorAll(".btn-pop");
 
-var page = document.getElementsByTagName('body')[0];
-
+var page = document.getElementsByTagName("body")[0];
+var popup = document.getElementsByClassName("desk-work-list");
 
 function displaymobilemenu() {
   //alert("changed");
   mobileMenu.classList.replace("mobile-menu-hide", "mobile-menu-show");
-  page.classList.add('noscroll');
+  page.classList.add("noscroll");
 }
 
 function hidemobilemenu() {
   //alert("changed");
```

Comment:
- [ ] as the previous reviewer asked it would be better to remove all the unused commented code to keep you file clean and professional

Reply:
I am Really sry .. done that :)